Network/Attributes/PacketTypeAttribute.cs

```diff
@@ -36,30 +41,123 @@ namespace Network.Attributes
-    /// - (3)   EstablishUdpRequest
-    /// - (4)   EstablishUdpResponse
-    /// - (5)   EstablishUdpResponseACK
-    /// - (6)   VersionRequest
-    /// - (7)   VersionResponse
-    /// - (8)   AddPacketTypeRequest
-    /// - (9)   AddPacketTypeResponse
-    /// - (10)  UDPPingRequest
-    /// - (11)  UDPPingResponse
-    /// Knowledge about the ID isn't essential anymore. (Since version 2.0.0.0)
-    /// Just do not overwrite the existing IDs above.
+    /// </description>
+    /// </listheader>
+    /// <item>
+    /// <term>0</term>
+    /// <description>
+    /// <see cref="PingRequest"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>1</term>
+    /// <description>
+    /// <see cref="PingResponse"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>2</term>
+    /// <description>
+    /// <see cref="CloseRequest"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>3</term>
+    /// <description>
+    /// <see cref="EstablishUdpRequest"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>4</term>
+    /// <description>
+    /// <see cref="EstablishUdpResponse"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>5</term>
+    /// <description>
+    /// <see cref="EstablishUdpResponseACK"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>6</term>
+    /// <description>
+    /// <see cref="AddPacketTypeRequest"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>7</term>
+    /// <description>
+    /// <see cref="AddPacketTypeResponse"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>8</term>
+    /// <description>
+    /// <see cref="UDPPingRequest"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>9</term>
+    /// <description>
+    /// <see cref="UDPPingResponse"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>10</term>
+    /// <description>
+    /// <see cref="RawData"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>11</term>
+    /// <description>
+    /// <see cref="RSAKeyInformationRequest"/>
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <term>12</term>
+    /// <description>
+    /// <see cref="RSAKeyInformationResponse"/>
+    /// </description>
+    /// </item>
+    /// </list>
     /// </summary>
+    /// <remarks>
+    /// Knowledge about the ID isn't essential anymore (Since version 2.0.0.0).
+    /// However, the above IDs should NOT be overwritten, for compatibility
+    /// purposes.
+    /// </remarks>
     [AttributeUsage(AttributeTargets.Class)]
     public class PacketTypeAttribute : Attribute
     {
-        public PacketTypeAttribute(UInt16 packetType)
+        #region Properties
+
+        /// <summary>
+        /// The ID to use for the decorated <see cref="Packet"/>.
+        /// </summary>
+        public ushort Id { get; }
```

Comment:
Add private set;

Reply:
Its a readonly property (set once and never again) so no private set is necessary. I checked this before making the change and see no reason to go back, as the set is never used again. Thus, a single call in the constructor suffices, and no 'private set' accessor is needed.